javascript/src/main/services/scheduleAPI.js

```diff
@@ -0,0 +1,75 @@
+import { fetchWithToken } from "main/utils/fetch"
+
+
+
+//schedule
+const fetchcreateScheduleJSON = async (schedule, getToken, onSuccess, onError) => {
```

Comment:
The prefix `fetch` and suffix `JSON` on each of these functions seems unnecessary and is out of step with the rest of the code base.  I wonder if we could just call each of these `createSchedule`, `deleteSchedule`, `getSchedule` and `getSchedules`?

Reply:
Otherwise, looks good to me!

Reply:
@andyyyho I saw you put in an approval, but didn't resolve this conversation. Can you (or another team member) let us know whether you plan to make this change? And if not, could you fix the name to be correctly camelCased?

Reply:
@andrewhlu Sorry about that! I just fixed the camel casing and pushed it to this branch.